### PR TITLE
Ft registration checker 160195781

### PR DESCRIPTION
**Pull request by kafuuma**
merge branch ft-registration-checker-160195781 to develop

### CODE REVIEW

**Comment by Luleherll** [comment]
it's ok

File: registration-checker/reception/reception.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ord_list =ord_file.readlines()
+for name in ord_list:
+    ord_guests.append(name.rstrip("\n"))
+
+##close files
+vip_file.close()
+ord_file.close()
+
+
+def search_vip_guest(name):
```
Comment (jeanjoe) [inline registration-checker/reception/reception.py]: The instruction says

`Name your main function registration_checker, its argument/input should be a single name`

you might need to create one function and and just make a call to it when you are searching the name from either VIP or ORDINARY

**Comment by KicaRonaldOkello** [comment]
Good concept, but rename your function to registration checker

**Comment by kaggwachristopher** [comment]
remove that comment on line 15 in reception. py and put it In the right place

**Pull request merged**